kestra-io/plugin-jira
# code(): add create comment on jira tickets task

This PR
* Supports a task creating a comment on a Jira ticket, generally this task can be used on task failure like the example below.
* QA Flow Config
```yaml
id: unreliable_flow
namespace: company.team

tasks:
  - id: xfail_task
    type: io.kestra.plugin.core.log.Log
    message: Hello {{outputs.username}}! 🚀 # Notice there is no such thing in outputs

errors:
  - id: alert_on_fail
    type: io.kestra.plugin.jira.issues.Create
    baseUrl: https://job-harness.atlassian.net
    username: "{{ secret('EMAIL'') }}"
    password: "{{ secret('API_TOKEN'') }}"
    projectKey: KJP
    summary: "{{ execution.id }} has failed on {{ taskrun.startDate }}"
    description: "{{ outputs['xfail_task'] }}"
    labels:
      - bug
      - workflow

  - id: coment_on_ticket
    type: io.kestra.plugin.jira.issues.CreateComment
    baseUrl: https://job-harness.atlassian.net
    username: "{{ secret('EMAIL'') }}"
    password: "{{ secret('API_TOKEN'') }}"
    projectKey: KJP
    issueIdOrKey: KJP-25
    body: "{{ execution.id }} has failed on {{ taskrun.startDate }}"
```
* Result
![Capture d'écran 2024-07-18 092411](https://github.com/user-attachments/assets/83c67845-d1a2-4f39-ac3f-ce48cd91d7fe)
![Capture d'écran 2024-07-18 092453](https://github.com/user-attachments/assets/ce547185-ddfa-46c5-8b0c-e85fbfff1df6)
![Capture d'écran 2024-07-18 092624](https://github.com/user-attachments/assets/82d75b5a-0308-4b6a-8b46-ac723cb209e4)
